app/lib/silas/oidc_strategy.rb

```diff
@@ -10,7 +10,9 @@ def office_codes
       return laa_accounts unless FeatureFlags.provider_data_api.enabled?
 
       ProviderDataApi::ActiveOfficeCodesFilter.call(
-        laa_accounts, area_of_law: 'CRIME LOWER'
+        laa_accounts,
+        area_of_law: 'CRIME LOWER',
+        translator: Providers::SchedulesToOfficeTranslator
```

Comment:
☝️ specifies Crime Apply custom translator instead of relying on PDAs determination of active.